### PR TITLE
Add basic portfolio page

### PR DESCRIPTION
**Pull request by ash-11**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,115 @@
...
+                <p>A Machine learning problem on Kaggle in which you have to predict which passengers survived the tragedy based on Age, Sex, Cabin Number, Embarked point, etc.</p>
+
+            </div>
+
+            <div class="por">
+                <h2>Positions of Responsibility</h2>
+                <ul>
+                    <li><b>Competitive Programming Head,</b> Coding Club IIT-Guwahati(2019-2020).
+                    <li><b>Problem Setter</b> of Coding contest in Kriti and Codejunk 2019 by Techniche(IIT-Guwahati).
+                    <li><b>Technothlon 2018:</b> City Representative of city Rudrapur(Uttarakhand).
```
Comment (rahulnagurtha) [inline portfolio/src/main/webapp/index.html:108]: Cliched POR for everyone at IITG :P

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,115 @@
...
+        <script src="script.js"></script>
+    </head>
+
+    <body>
+
+        <div id="content">
+            <h1 class="center">Ashish Agarwal</h1>
+            <p class="center">ashish01012000@gmail.com | 844.782.2348 | ashis170123011@iitg.ac.in</p>
+            <p>I'm a final year B.Tech(Mathematics and Computing) student at IIT-Guwahati.
+                I spend most of my free time in competitive programming and watching animes, web series.
```
Comment (rahulnagurtha) [inline portfolio/src/main/webapp/index.html:17]: You should suggest me more web series regarding history, strategy and economics.

**Review by rahulnagurtha — APPROVED** [review]

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (ash-11) [reply portfolio/src/main/webapp/index.html:17]: House of Cards is a really good combination strategy and politics. I don't know much about history or economics. 
I am more of an Anime guy. Deathnote is definitely worth watching if you like strategy.

Comment (rahulnagurtha) [reply portfolio/src/main/webapp/index.html:17]: Sure, thanks.

**Pull request merged**